clients/src/test/java/org/apache/kafka/common/config/ConfigDefTest.java

```diff
@@ -722,4 +724,35 @@ public void testNiceTimeUnits() {
         assertEquals(" (365 days)", ConfigDef.niceTimeUnits(Duration.ofDays(365).toMillis()));
     }
 
+    @Test
+    public void testThrowsExceptionWhenListSizeExceedsLimit() {
+        assertThrows(ConfigException.class, () -> new ConfigDef().define("lst",
+                                                                         Type.LIST,
+                                                                         asList("a", "b"),
+                                                                         ListSize.atMostOfSize(1),
+                                                                         Importance.HIGH,
+                                                                         "lst doc"));
+    }
+
+    @Test
+    public void testNoExceptionIsThrownWhenListSizeEqualsTheLimit() {
+        final List<String> lst = asList("a", "b", "c");
+        new ConfigDef().define("lst",
```

Comment:
We can use `assertDoesNotThrow` here and below.

Reply:
Done